### PR TITLE
Added and updated tools and repos

### PR DESCRIPTION
**Pull request by stuhli**
I have added some tools and other repos. I have tried to make sure that these sources have been updated at least within the last six months and have some thematic relevance to the work of incident response folks. I also took the liberty of making some updates and slightly adjusting the structure in some places.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,11 +109,24 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
...
+* [KAPE](https://www.kroll.com/en/services/cyber-risk/investigate-and-respond/kroll-artifact-parser-extractor-kape) - A triage tool that finds the most prevalent digital artifacts and then parses them quickly. Great and thorough when time is of the essence.
 * [RTIR](https://www.bestpractical.com/rtir/) - Request Tracker for Incident Response (RTIR) is the premier open source incident handling system targeted for computer security teams. We worked with over a dozen CERT and CSIRT teams around the world to help you handle the ever-increasing volume of incident reports. RTIR builds on all the features of Request Tracker.
 * [Sandia Cyber Omni Tracker (SCOT)](https://github.com/sandialabs/scot) - Incident Response collaboration and knowledge capture tool focused on flexibility and ease of use. Our goal is to add value to the incident response process without burdening the user.
 * [threat_note](https://github.com/defpoint/threat_note) - Lightweight investigation notebook that allows security researchers the ability to register and retrieve indicators related to their research.
 
+### IOC Management
```
Comment (meirwah) [inline README.md:117]: let's remove this section, there are other lists covering this like https://github.com/hslatman/awesome-threat-intelligence

File: README.md [hunk 2]
```diff
@@ -205,13 +227,18 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
...
 * [Phantom Community Playbooks](https://github.com/phantomcyber/playbooks) - Phantom Community Playbooks for Splunk but also customizable for other use.
+* [ThreatHunter-Playbook](https://github.com/OTRF/ThreatHunter-Playbook) - Playbook to aid the development of techniques and hypothesis for hunting campaigns.
 
 ### Process Dump Tools
 
 * [Microsoft ProcDump](https://docs.microsoft.com/en-us/sysinternals/downloads/procdump) - Dumps any running Win32 processes memory image on the fly.
 * [PMDump](http://www.ntsecurity.nu/toolbox/pmdump/) - Tool that lets you dump the memory contents of a process to a file without stopping the process.
 
-### Sandboxing/reversing tools
+### Ransomware Mitigation Tools
```
Comment (meirwah) [inline README.md:237]: let's not add a section with so little in it

File: README.md [hunk 3]
```diff
@@ -220,28 +247,46 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
...
 * [Yomi](https://yomi.yoroi.company) - Free MultiSandbox managed and hosted by Yoroi.
 
-### Timeline tools
+### Scanner Tools
+
+* [Fenrir](https://github.com/Neo23x0/Fenrir) - Simple IOC scanner. It allows scanning any Linux/Unix/OSX system for IOCs in plain bash. Created by the creators of THOR and LOKI.
+* [LOKI](https://github.com/Neo23x0/Loki) - Free IR scanner for scanning endpoint with yara rules and other indicators(IOCs).
+* [Spyre](https://github.com/spyre-project/spyre) - Simple YARA-based IOC scanner written in Go
+
+### Security Orchestration, Automation and Response (SOAR)
```
Comment (meirwah) [inline README.md:272]: these should go https://github.com/meirwah/awesome-incident-response#incident-management

File: README.md [hunk 4]
```diff
@@ -220,28 +247,46 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
...
+
+* [DFTimewolf](https://github.com/log2timeline/dftimewolf) - A framework for orchestrating forensic collection, processing and data export.
+* [Shuffle](https://github.com/frikky/Shuffle) - A general purpose security automation platform focused on accessibility.
+
+### Sysmon Config Templates
```
Comment (meirwah) [inline README.md:277]: let's not add this one too, not much in it

File: README.md [hunk 5]
```diff
@@ -220,28 +247,46 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
...
+* [Rizin](https://github.com/rizinorg/rizin) - UNIX-like reverse engineering framework and command-line toolset
 * [StringSifter](https://github.com/fireeye/stringsifter) - A machine learning tool that ranks strings based on their relevance for malware analysis.
 * [Valkyrie Comodo](https://valkyrie.comodo.com) - Valkyrie uses run-time behavior and hundreds of features from a file to perform analysis.
 * [Viper](https://github.com/viper-framework/viper) - Python based binary analysis and management framework, that works well with Cuckoo and YARA.
 * [Virustotal](https://www.virustotal.com) - Free online service that analyzes files and URLs enabling the identification of viruses, worms, trojans and other kinds of malicious content detected by antivirus engines and website scanners.
 * [Visualize_Logs](https://github.com/keithjjones/visualize_logs) - Open source visualization library and command line tools for logs (Cuckoo, Procmon, more to come).
 * [Yomi](https://yomi.yoroi.company) - Free MultiSandbox managed and hosted by Yoroi.
 
-### Timeline tools
+### Scanner Tools
```
Comment (meirwah) [inline README.md:266]: not sure this section makes sense here

**Review by meirwah — COMMENTED** [review]
plz for next time, don't make 1 PR with many changes
split PR to smaller PR's , so it will easier and faster to review

**Comment by stuhli** [comment]
Sorry for making you wait so long. Yes, I understand, next time I'll split it.

I will look at the comments and implement. If the merge becomes too confusing due to the long time, I'll close the PR and start over individually. Then also individually.

File: README.md (hunk 1 above)
Comment (stuhli) [reply README.md]: done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d

File: README.md (hunk 2 above)
Comment (stuhli) [reply README.md]: done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d
moved to https://github.com/meirwah/awesome-incident-response#other-tools

File: README.md (hunk 3 above)
Comment (stuhli) [reply README.md]: done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d

File: README.md (hunk 4 above)
Comment (stuhli) [reply README.md]: done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d
moved to https://github.com/meirwah/awesome-incident-response#other-tools

File: README.md (hunk 5 above)
Comment (stuhli) [reply README.md:260]: Unfortunately, I can't think of another section where I could simply move it.

In the incident case, being able to scan systems for IOCs with a stand-alone tool (without EDR) is, in my view, an important task that has to be mastered in IR and which I would therefore look for here in the list. But I am also happy to be convinced of the opposite.

**Comment by stuhli** [comment]
I'm through with my changes for now.

I have implemented your comments - with one exception (see above at the code snippets for details). I have merged the current state from the master and removed the conflict. I also corrected a few small things in the table of contents.

Have a look at it when you get the chance, if it is ok.

**Comment by meirwah** [comment]
hey
sorry for getting late on this

Can you update your branch from master ?

**Comment by stuhli** [comment]
Hi,
I hope I could solve the conflicts correctly?!

**Comment by meirwah** [comment]
will look, for future, plz break your changes to smaller PR's which are much easier to review and merge

File: README.md [hunk 6]
```diff
@@ -129,13 +143,15 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 * [AppCompatProcessor](https://github.com/mbevilacqua/appcompatprocessor) - AppCompatProcessor has been designed to extract additional value from enterprise-wide AppCompat / AmCache data beyond the classic stacking and grepping techniques.
 * [APT Hunter](https://github.com/ahmedkhlief/APT-Hunter) - APT-Hunter is Threat Hunting tool for windows event logs.
+* [Beagle](https://github.com/yampelo/beagle) - Incident response and digital forensics tool which transforms security logs and data into graphs.
```
Comment (meirwah) [inline README.md:146]: this one doesn't seem maintained anymore, remove it plz

Comment (stuhli) [reply README.md]: done with f7d79d80d81a96c63c43dde7555ef47ec235f748

**Review by meirwah — APPROVED** [review]

**Pull request merged**